src/ashell/shell-state.c

```diff
@@ -589,6 +617,7 @@ static const struct ashell_cmd commands[] =
     ASHELL_COMMAND("run",   "[FILE] Runs the JavaScript program in the file" ,ashell_run_javascript),
     ASHELL_COMMAND("parse", "[FILE] Check if the JS syntax is correct"       ,ashell_parse_javascript),
     ASHELL_COMMAND("stop",  "Stops current JavaScript execution"             ,ashell_stop_javascript),
+    ASHELL_COMMAND("cfg",   "[FILE] Set the file that should run at boot"    ,ashell_set_bootcfg),
```

Comment:
should we rename this to something easier to understand? like "boot", "autorun"?, cfg just not easy to understand.